.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: bug
+assignees: karasusan
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+If you want to ask the question, you can post the message on the [Unity Forum](https://forum.unity.com/threads/unity-render-streaming-introduction-faq.742481/).
```

Comment:
Isn't github issue better ?

Reply:
Reporting bugs and feature requests should be posted on Github Issue. On the other hand, I guess it is appropriate to collect general questions on the Unity Forum.